packages/shared/lib/core/wallet/stores/mint-token-details.store.ts

```diff
@@ -0,0 +1,30 @@
+import { writable } from 'svelte/store'
+import { IMintTokenDetails } from '../interfaces'
+
+export const mintTokenDetails = writable<IMintTokenDetails>({
+    name: undefined,
+    totalSupply: undefined,
+    circulatingSupply: undefined,
+    decimals: 0,
+    symbol: undefined,
+    description: undefined,
+    url: undefined,
+    logoUrl: undefined,
+})
+
+export function updateMintTokenDetails(payload: IMintTokenDetails): void {
```

Comment:
This should either be called `setMintTokenDetails` or have the logic so that it updates including the existing state rather than setting it.